packages/webservice/app/src/main/resources/db/changelog/changelog.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="admin" id="1539122075">
+        <sql>
+          CREATE TABLE AsyncQuery (
+              id varchar(36) NOT NULL,
+              asyncAfterSeconds int,
+              requestId varchar(36),
+              principalName varchar(255),
+              query varchar(1024) NOT NULL,
```

Comment:
Should this be larger (big in list filters)?

Reply:
changed to 20K